firstvoices/backend/tasks/visibility_tasks.py

```diff
@@ -44,27 +51,25 @@ def bulk_visibility_change_job(job_instance_id):
             story_pages.update(visibility=job.to_visibility)
             pages.update(visibility=job.to_visibility)
             widgets.update(visibility=job.to_visibility)
+            # Change Site visibility
+            site.visibility = job.to_visibility
+            site.save()
 
-        # Change Site visibility
-        site.visibility = job.to_visibility
-        site.save()
     except Exception as e:
         # if there’s an error during the updates:
         # add the message to the batch visibility job instance and set the status to failed
         job.status = JobStatus.FAILED
         job.message = str(e)
         job.save()
 
-        indexing_paused = site.sitefeature_set.get(key="indexing_paused")
-        indexing_paused.is_enabled = False
-        indexing_paused.save()
+        indexing_paused_feature[0].is_enabled = False
+        indexing_paused_feature[0].save()
         return
 
     # Resume search indexing for site, + reindex entire site
     sync_all_site_content_in_indexes(site)
-    indexing_paused = site.sitefeature_set.get(key="indexing_paused")
-    indexing_paused.is_enabled = False
-    indexing_paused.save()
+    indexing_paused_feature[0].is_enabled = False
+    indexing_paused_feature[0].save()
```

Comment:
This is minor, but should we turn indexing back on before we run the full re-index? The re-indexing takes some time, and any edits made during that time would not get synced in the index.